packages/interface/src/components/admin-menu-toggle/index.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { Button } from '@wordpress/components';
+import { useRef, useEffect, useState } from '@wordpress/element';
+import { ESCAPE, TAB } from '@wordpress/keycodes';
+import { __ } from '@wordpress/i18n';
+import { wordpress } from '@wordpress/icons';
+
+function AdminMenuToggle() {
+	const buttonRef = useRef();
+	const toggleMenu = useToggle( { ref: buttonRef } );
+
+	const { isActive, postType } = useSelect( ( select ) => {
+		const { getCurrentPostType } = select( 'core/editor' );
+		const { isFeatureActive } = select( 'core/edit-post' );
```

Comment:
This component has been moved to a generic package "interface" which means it can't use a specific store like "core/edit-post" or "core/editor". I'm not even certain why we're checking here the post type here. For me we should remove all that use select code and the check to return null a bit under.

We should just not render this component from its parent when we're not on fullscreen.

Reply:
Took me a bit to understand this, but now it makes sense. I've removed the display logic out of the component and instead moved it into where it's used.